static/scss/bound_check_box.scss

```diff
@@ -13,9 +13,12 @@
     }
 
     .label {
-      font-weight: 500;
+      font-weight: 300;
       font-size: 14px;
-      color: $font-black;
+      color: $medlightgrey;
+      position: relative;
```

Comment:
This is the text for agreeing to the Terms of Service in the sign up flow. The relative positioning aligns it better with the checkbox image.